package.json

```
@@ -13,9 +13,9 @@
     "gulp-sass": "^4.0.2",
     "gulp-sourcemaps": "^3.0.0",
     "gulp-util": "^3.0.8",
+    "uswds": "^2.10.0",
     "snyk": "^1.437.3",
     "npm": "^6.14.10",
```

@h-m-f-t, the extra trailing comma on your (unalphabetized) `npm` line is causing the issue.  This should fix it.
```suggestion
    "npm": "^6.14.10",
    "snyk": "^1.437.3",
    "uswds": "^2.10.0"
```

---

Yep, I see where I went awry. Resolving conflicts in the web editor is tricky. Thank you, @jsf9k!

---

> Yep, I see where I went awry. Resolving conflicts in the web editor is tricky. Thank you, @jsf9k!

No problem.  JSON is finicky about the last entry in a list _not_ having a comma.  I've been bitten by this before too. :vampire_man: